script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_5B.txt

```diff
@@ -98,55 +98,55 @@ All I want right now is to be left alone in my room, take off my glasses, close
 @gThe woman swimming in the crowd is beautiful, both pure and desirable.
 <Page20>
 "―――――――"
-@gMy throat is aching.
-@gI haven't been able to breathe properly for a while now.
+@gMy THROAT is ACHING.//@g　ノドがアツイ。katakana
+@gI haven't been ABLE to BREATHE for a WHILE now.//@g　サッキからイキがぜんぜんデキナイ。
```

Comment:
Maybe mark both of these in some way so we can improve once we have better font rendering